caproto/threading/client.py

```diff
@@ -833,7 +833,12 @@ def get(self, *, count=None, as_string=False, as_numpy=True,
             return info['char_value']
         elif as_string and self.typefull in ca.enum_types:
             enum_strs = self.enum_strs
-            ret = [enum_strs[r] for r in info['value']]
+            ret = []
```

Comment:
Can you add a comment on what we are silently ignoring here?

Reply:
It's not impossible to have invalid enum values that don't correspond to any string (I think that's what @tacaswell ran into here)